sycl/test/basic_tests/buffer/buffer.cpp

```diff
@@ -1,3 +1,5 @@
+// REQUIRES: opencl-interop
+
```

Comment:
Should we remove "XFAIL: cuda" below?
BTW, different tests handle tests like this differently:
- `XFAIL: cuda`
- `UNSUPPOTED: cuda`
- `REQUIRES: opencl`